colcon_mixin/mixin/mixin_argument.py

```diff
@@ -149,8 +153,11 @@ def collect_parsers_by_verb(root, parsers, parent_verbs=()):
                 mixin_arguments[verb] = self._add_mixin_argument(
                     p, groups[p], verb)
 
-        with SuppressUsageOutput([self._parser] + list(parsers.values())):
-            known_args, _ = self._parser.parse_known_args(*args, **kwargs)
+        parsers_to_suppress = [self._parser] + list(parsers.values())
+        types_to_omit = [_argparse_existing_file]
```

Comment:
Why is this type conversion omitted from the type suppression?

Depending on the answer it may make sense to leave a comment with the explanation.

Reply:
That type is actually used for taking action based on the preliminary parsing operation, and was added by this file.
Added here: https://github.com/colcon/colcon-mixin/blob/b11b7a6fa1e90646c2bfa3478bb0408556ca2fe6/colcon_mixin/mixin/mixin_argument.py#L197-L200
Used here (prior to 'final' parsing): https://github.com/colcon/colcon-mixin/blob/b11b7a6fa1e90646c2bfa3478bb0408556ca2fe6/colcon_mixin/mixin/mixin_argument.py#L162-L164

Reply:
Got it. Thanks. I think a comment could be useful but if I were coming back to look at this again I would probably grep around for the omitted type. In this case I would find it nearby.